src/components/store/Likes.js

```diff
@@ -0,0 +1,41 @@
+import { createContext, useState } from 'react';
+
+const LikesContext = createContext({
+    likes: [],
+    addLike: (like) => {console.log("here")},
+    removeLike: (likeID) => {console.log("remove here")},
```

Comment:
Are these logs placeholders? If so, it should have a todo comment:
// TODO: placeholders for adding and removing likes

Or better, to implement it as a part of the feature

Reply:
Yea was meant to be more for auto-fill purposes to view what objects are in the context provider function - have removed the console.log and commented on them

Reply:
Coming from Java, I like adding a prefix like // TODO: comment
This is because modern IDEs can keep track of these todo comments and show it you in one place for addressing.
Even outside of Java I use it, because then I can just search for the prefix agreed by the team (like TODO) and address these comments. You risk leaving legacy comments lost in the code base forever without it. Part of the reason why people dislike adding TODOs especially for things that can be easily implemented